### ISSUE TITLE
Browsing to Workers Page fails with Internal Server Error

### ISSUE THREAD

**Issue by rbollar** [guydavis/machinaris#429]
**Describe the bug**

This Machinaris installation is in a state where the /plotting/workers page will not display. It returns an Internal Server Error. It appears to be a parsing error:

```
2021-11-19 08:20:54 [118] [ERROR] Exception on /plotting/workers [GET]
Traceback (most recent call last):
  File "/chia-blockchain/venv/lib/python3.9/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/chia-blockchain/venv/lib/python3.9/site-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/chia-blockchain/venv/lib/python3.9/site-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/chia-blockchain/venv/lib/python3.9/site-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/machinaris/web/routes.py", line 112, in plotting_workers
    disk_usage = stats.load_recent_disk_usage('plotting')
  File "/machinaris/web/actions/stats.py", line 177, in load_recent_disk_usage
    path_values.append(paths[path][date])
KeyError: '2021-11-18T0950'

```

**To Reproduce**

Unfortunately, I don't have reproduction steps.

**Expected behavior**

/plotting/workers page to display.

**System setup:**
 - OS: TrueNAS SCALE RC1.1
 - Docker version number: N/A
 - Machinaris branch: main
 - Machinaris version number 0.6.4

**Comment by guydavis** [comment]
Welcome!  Thanks for providing a detailed log like above.  Very refreshing to see a detailed error report!  Sorry you are encountering this on TrueNAS with Machinaris v0.6.4.

Please [update](https://github.com/guydavis/machinaris/wiki/Releases#upgrading) to the latest version (just released in past hour): version 0.6.5.  In particular, I added a [guard](https://github.com/guydavis/machinaris/blame/main/web/actions/stats.py#L179) in the code for this error.  Hope this  helps!

**Comment by rbollar** [comment]
Very good! I can see the page now.
